<launch>
    <arg name="model" />
  
    <param name="robot_description"
      command="$(find xacro)/xacro --inorder $(find  conveyor_description_pkg)/urdf/conveyor.xacro" />
    <param name="use_gui" value="true" />
  
    <!-- LIDAR -->
    <!-- <node pkg="ld08_driver" type="ld08_driver" name="turtlebot3_lds" output="screen">
      <param name="port" value="/dev/ttyUSB0" />
      <param name="frame_id" value="hokuyo_joint" />
    </node> -->
  
  
    
    <group ns="robot1">
        <!-- Algorithm -->
        <node name="model_mapping" pkg="conveyor_description_pkg" type="mapping" />
    
        <!-- <node name="arduino2rviz" pkg="conveyor_description_pkg" type="arduino2rviz" /> -->
        <node name="serial_node" pkg="rosserial_python" type="serial_node.py">
        <param name="port" type="string" value="/dev/ttyACM0" />
        <param name="baud" type="int" value="1000000" />
        </node>
    </group>

  <!-- 
    <node name="joint_state_publisher" type="joint_state_publisher" pkg="joint_state_publisher"
      output="screen">
      <rosparam param="source_list">[to_joint_states]</rosparam>
    </node> 
    
  
    <node name="rviz" pkg="rviz" type="rviz"
      args="-d $(find conveyor_description_pkg)/urdf/conveyor.rviz" required="true" /> -->
  
  
  </launch>